shared/modules/object.utils.js

```diff
@@ -18,7 +18,7 @@ export const AllProperties = Symbol('*');
  * If a property is excluded, its type is included instead.
  *
  * @param {object} object - The object to mask
- * @param {Object<object | boolean>} mask - The mask to apply to the object
+ * @param {{[key: string]: object | boolean}} mask - The mask to apply to the object
```

Comment:
This is one of the breaking changes in eslint jsdoc. `Object` syntax isn't permitted anymore.